src/GraphQL.Conventions/Web/RequestHandler.cs

```diff
@@ -37,12 +37,20 @@ internal RequestHandlerBuilder()
                 _serviceProvider = this;
             }
 
+            public RequestHandlerBuilder WithServiceProvider(IServiceProvider serviceProvider)
+            {
+                _serviceProvider = serviceProvider;
+                return this;
+            }
+
+            [Obsolete("Please use WithServiceProvider instead.")]
             public RequestHandlerBuilder WithDependencyInjector(IServiceProvider serviceProvider)
```

Comment:
Let's remove this overload.  The argument will be wrong anyway, and VS will match on the other overload and still show the obsolete warning.